controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -517,6 +551,14 @@ func (r *OCSInitializationReconciler) ensureOcsOperatorConfigExists(initialData
 		util.DisableCSIDriverKey:         strconv.FormatBool(true),
 	}
 
+	if util.EnableTopologyKey == "true" {
```

Comment:
If you want to compare to boolean as a string, please use parse bool instead of comparing to a hardcoded string value
```suggestion
	if val. err := strconv.ParseBool(util.EnableTopologyKey; err == nil && val  {
```

Reply:
util.EnableTopologyKey is a constant with the value of "CSI_ENABLE_TOPOLOGY" I think you need to call the function `getEnableTopologyKeyValue` or read it from the configmap data